tre_schemas/dri-preingest-sip-error.json

```diff
@@ -0,0 +1,46 @@
+{
+    "$schema": "http://json-schema.org/draft-04/schema",
+    "$id":"https://nationalarchives.gov.uk/da-transform/tre/schemas/dri-preingest-sip-available",
+    "$ref":"https://nationalarchives.gov.uk/da-transform/tre/schemas/tre-event",
+    "type": "object",
+    "properties": {
+        "producer": {
+            "properties": {
```

Comment:
I note [bagit-validation-error.json](https://github.com/nationalarchives/da-transform-schemas/blob/main/tre_schemas/bagit-validation-error.json) included the following line after this point - I'm not sure it matters but thought it best to flag it:

```"type": "object",```